diff --git a/node_modules/@base-ui-components/react/dialog/store/DialogStore.js b/node_modules/@base-ui-components/react/dialog/store/DialogStore.js
index 68156b2..e652802 100644
--- a/node_modules/@base-ui-components/react/dialog/store/DialogStore.js
+++ b/node_modules/@base-ui-components/react/dialog/store/DialogStore.js
@@ -71,7 +71,6 @@ function createInitialState(initialState = {}) {
     ...(0, _popups.createInitialPopupStoreState)(),
     modal: true,
     disablePointerDismissal: false,
-    popupElement: null,
     viewportElement: null,
     descriptionElementId: undefined,
     titleElementId: undefined,
diff --git a/node_modules/@base-ui-components/react/esm/dialog/store/DialogStore.js b/node_modules/@base-ui-components/react/esm/dialog/store/DialogStore.js
index 08f4672..a0e23a7 100644
--- a/node_modules/@base-ui-components/react/esm/dialog/store/DialogStore.js
+++ b/node_modules/@base-ui-components/react/esm/dialog/store/DialogStore.js
@@ -63,7 +63,6 @@ function createInitialState(initialState = {}) {
     ...createInitialPopupStoreState(),
     modal: true,
     disablePointerDismissal: false,
-    popupElement: null,
     viewportElement: null,
     descriptionElementId: undefined,
     titleElementId: undefined,
